### PR TITLE
Various gRPC doc fixes and improvements

### PR DESCRIPTION
**Pull request by JamesNK**

### CODE REVIEW

File: aspnetcore/grpc/test-tools.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: Test gRPC services with gRPCurl in ASP.NET Core
 author: jamesnk
 description: Learn how to test services with gRPC tools. gRPCurl a command-line tool for interacting with gRPC services. gRPCui is an interactive web UI.
-monikerRange: '>= aspnetcore-3.1'
+monikerRange: '>= aspnetcore-3.0'
```
Comment (JamesNK) [inline aspnetcore/grpc/test-tools.md:5]: I'm not sure why this was updated to 3.1. gRPC supports 3.0 and above.

File: aspnetcore/grpc/comparison.md [hunk 2]
```diff
@@ -79,19 +79,23 @@ gRPC is well suited to the following scenarios:
...
+There are two common approaches to bring gRPC to browser apps:
 
-Not all of gRPC's features are supported by gRPC-Web. Client and bi-directional streaming isn't supported, and there is limited support for server streaming.
+* [gRPC-Web](https://grpc.io/docs/tutorials/basic/web.html) is an additional technology from the gRPC team that provides gRPC support in the browser. gRPC-Web allows browser apps to benefit from the high-performance and low network usage of gRPC. Not all of gRPC's features are supported by gRPC-Web. Client and bi-directional streaming isn't supported, and there is limited support for server streaming.
 
-> [!TIP]
-> .NET Core has support for gRPC-Web. Visit <xref:grpc/browser> for more information.
+  .NET Core has support for gRPC-Web. Visit <xref:grpc/browser> for more information.
+
+* RESTful JSON Web APIs can be automatically created from gRPC services by annotating the *.proto* with [HTTP metadata](https://cloud.google.com/service-infrastructure/docs/service-management/reference/rpc/google.api#google.api.HttpRule). This allows an app to support both gRPC and JSON Web APIs, without duplicating effort of building separate services for both.
```
Comment (scottaddie) [inline aspnetcore/grpc/comparison.md:96]:
```suggestion
* RESTful JSON Web APIs can be automatically created from gRPC services by annotating the *.proto* file with [HTTP metadata](https://cloud.google.com/service-infrastructure/docs/service-management/reference/rpc/google.api#google.api.HttpRule). This allows an app to support both gRPC and JSON web APIs, without duplicating effort of building separate services for both.
```

File: aspnetcore/grpc/comparison.md [hunk 3]
```diff
@@ -79,19 +79,23 @@ gRPC is well suited to the following scenarios:
...
+
+  .NET Core has experimental support for creating JSON Web APIs from gRPC services. Visit <xref:grpc/httpapi> for more information.
```
Comment (scottaddie) [inline aspnetcore/grpc/comparison.md:98]:
```suggestion
  .NET Core has experimental support for creating JSON web APIs from gRPC services. For more information, see <xref:grpc/httpapi>.
```

**Review by scottaddie — APPROVED** [review]
A few minor changes to consider, and then this looks good.

File: aspnetcore/grpc/comparison.md [hunk 4]
```diff
@@ -79,19 +79,23 @@ gRPC is well suited to the following scenarios:
...
+There are two common approaches to bring gRPC to browser apps:
 
-Not all of gRPC's features are supported by gRPC-Web. Client and bi-directional streaming isn't supported, and there is limited support for server streaming.
+* [gRPC-Web](https://grpc.io/docs/tutorials/basic/web.html) is an additional technology from the gRPC team that provides gRPC support in the browser. gRPC-Web allows browser apps to benefit from the high-performance and low network usage of gRPC. Not all of gRPC's features are supported by gRPC-Web. Client and bi-directional streaming isn't supported, and there is limited support for server streaming.
 
-> [!TIP]
-> .NET Core has support for gRPC-Web. Visit <xref:grpc/browser> for more information.
+  .NET Core has support for gRPC-Web. Visit <xref:grpc/browser> for more information.
```
Comment (JamesNK) [inline aspnetcore/grpc/comparison.md:94]:
```suggestion
  .NET Core has support for gRPC-Web. For more information, see <xref:grpc/browser>.
```

**Pull request merged**